src/Form/Type/UiElement/ButtonLinkType.php

```diff
@@ -41,8 +42,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                     'includeInternalMessages' => false,
                     'message' => 'monsieurbiz_richeditor_plugin.not_valid_url',
                     'constraints' => [
-                        new Assert\Url(['protocols' => ['http', 'https'], 'relativeProtocol' => true]),
-                        new Assert\Regex(['pattern' => '`^(#|/.*)$`']),
+                        new Assert\Regex(['pattern' => '`^(#|/|tel:|mailto:)`']),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** why don't you keep the contraint Url? ‏

Reply:
Yes, we could use `AtLeastOneOf` constraint to apply `Url` or `Regex`

https://symfony.com/doc/5.x/reference/constraints/AtLeastOneOf.html